install_bootstrap

```diff
@@ -249,7 +249,15 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
 #            read -ep $'What is the email address for notification? \n' email_address
 #      fi
 
-      
+      project_name='project_name'
```

Comment:
This will only come into play if there is an email_whitelist_name.

I think you should ask for the email_whitelist_name first. If the user does not enter one, then you shouldn't prompt for this nor for the email.

Reply:
Good point. I've reorganized.